packages/beagle-react/src/components/BeagleDynamicLists/DynamicListCore/index.tsx

```diff
@@ -77,30 +83,34 @@ const BeagleListView: FC<BeagleListViewInterface> = ({
           component.__suffix__ = `${suffix}:${iterationKey}`
         }
       })
-      
+
       return templateTree
     })
 
     viewContentManager.getView().getRenderer().doFullRender(element, element.id)
   }, [JSON.stringify(dataSource)])
 
   return (
-    <StyledListView
+    <StyledDynamicViewsInterface
       ref={elementRef}
       className={className}
       direction={direction}
       useParentScroll={useParentScroll}
       style={style}
-      isScrollIndicatorVisible = {isScrollIndicatorVisible}
+      isScrollIndicatorVisible={isScrollIndicatorVisible}
+      numColumns={numColumns}
+      listType={listType}
       {
-        ...({ [direction === 'VERTICAL' ? 
-          'aria-rowcount' : 'aria-colcount']: Children.count(children) || 0 })
+        ...({
+          [direction === 'VERTICAL' ?
+            'aria-rowcount' : 'aria-colcount']: Children.count(children) || 0,
+        })
```

Comment:
Based on the suggestion of using String Enums (down on this PR), I suggest:

```suggestion
          [direction === ListDirection.Vertical ?
            'aria-rowcount' : 'aria-colcount']: Children.count(children) || 0,
        })
```